packages/mux-uploader/src/mux-uploader-drop.ts

```diff
@@ -1,12 +1,30 @@
 import { globalThis, document } from './polyfills';
+import { getMuxUploaderEl } from './utils/element-utils';
+import type MuxUploaderElement from './mux-uploader';
 
 const template = document.createElement('template');
 
-/** @todo: Currently removing all styles. Follow up on overlay styling (CJP) */
-template.innerHTML = `
+template.innerHTML = /*html*/ `
```

Comment:
question(non-blocking): does this comment help with styling in your editor?

Reply:
yeah. i dislike it, but seems to help with syntax niceties in the editor